### ISSUE TITLE
CentOS 7 - No package Judy-devel available.

### ISSUE THREAD

**Issue by as1988** [netdata/netdata#9605]
#####Question summary

Hi, 
When I attempt to install via the kickstart method I get this error:

```
Press ENTER to run it >
yum install Judy-devel
Loaded plugins: fastestmirror, universal-hooks
Loading mirror speeds from cached hostfile
 * EA4: 91.197.228.252
 * cpanel-addons-production-feed: 91.197.228.252
 * cpanel-plugins: 91.197.228.252
 * base: mirrors.clouvider.net
 * extras: uk.mirror.nsec.pt
 * updates: mirror.sov.uk.goscomb.net
No package Judy-devel available.
Error: Nothing to do



We are very sorry!

Installation of required packages failed.

What to do now:

  1. Make sure your system is updated.
     Most of the times, updating your system will resolve the issue.

  2. If the error message is about a specific package, try removing
     that package from the command and run it again.
     Depending on the broken package, you may be able to continue.

  3. Let us know. We may be able to help.
     Open a github issue with the above log, at:

           https://github.com/netdata/netdata/issues


 FAILED

 WARNING  It failed to install all the required packages, but installation might still be possible.
```

It would appear the Judy is not available for install on my version of CentOS 7? How do I get around this?

Thank you!

##### OS / Environment
CentOS Linux release 7.8.2003 (Core) - 3.10.0-1062.9.1.el7.x86_64
cPanel v88.0.13

**Comment by Ferroin** [comment]
You appear to not have the EPEL repo enabled, which is required for installation to work correctly. The installation process _should_ try to install this itself, so you most likely said no to the prompt asking about installing it.

**Comment by as1988** [comment]
> You appear to not have the EPEL repo enabled, which is required for installation to work correctly. The installation process _should_ try to install this itself, so you most likely said no to the prompt asking about installing it.

It’s definitely installed, or at least I’ve ran the command for it to be installed and I’ve not said no to it being installed either. I presumably haven’t been asked as it too recognised that it is installed. Any ideas on how to resolve?

**Comment by prologic** [comment]
There are open issues with Judy and Judy-devel currently. I'm working on bundling the Judy development libraries into our source tree. @Ferroin Perhaps you can assist me with this?

**Comment by rdifrancesco** [comment]
@prologic Could you elaborate on what those issues are with Judy?  Could any of them be related to agents unable to connect to Netdata Cloud?

**Comment by as1988** [comment]
Hi
I have managed to resolve this issue.
You first need to:
yum remove epel-release

and then 
yum install epel-release

and then finally
yum install Judy-devel

and it actually locates the package and installs it!

**Comment by prologic** [comment]
That's kind of odd that removing a repository and re-adding it fixes this. But I'm glad it worked for you.

@rdifrancesco To answer your question we've run into a situation in general where Judy is harder to get on some systems, and on CentOS 8 for example there is a dependency conflict that I cannot seem to resolve easily.

So I'm trying to bundle the library into our source tree to avoid these problems in the first place...

**Comment by prologic** [comment]
This is being fixed by #9776
